tests/reframe/config/settings_magic_castle.py

```diff
@@ -0,0 +1,92 @@
+site_configuration = {
```

Comment:
let's then call this system magic castle and have the config sync with the deployment in EESSI, like partition names

Reply:
> have the config sync with the deployment in EESSI

Not sure what you mean with this. I'm not sure how 'standardized'the magic castle deployment was and if e.g. partition names will be the same next time. 

In any case, it _does_ serve as a 'neutral' (i.e. not related to a particular institute) example of a valid configuration file. I'll add some notes in the file to indicate that it is an example.